### PR TITLE
[Task] Daily Plan | add 'Remove from all plans' option to Action menu

### PR DESCRIPTION
**Pull request by Innocent-Akim**
[Task] 'Daily Plan' | add 'Remove from all plans' option to Action menu #2673

Users should be able to remove plans from a specific day as well as from multiple days
'Remove from all plans'

@evereq 
@Cedric921

### CODE REVIEW

File: apps/web/app/interfaces/IDailyPlan.ts [hunk 1]
```diff
@@ -8,6 +9,12 @@ export interface IDailyPlanBase extends IBasePerTenantAndOrganizationEntity {
 	status: DailyPlanStatusEnum;
 }
 
+export class IRemoveTaskFromManyPlans {
```
Comment (Cedric921) [inline apps/web/app/interfaces/IDailyPlan.ts:12]: Better to Use Interface instead of Class

**Comment by Cedric921** [comment]
https://github.com/user-attachments/assets/f1bc2744-0b7c-40a0-ad7d-896184212b11

File: apps/web/app/interfaces/IDailyPlan.ts (hunk 1 above)
Comment (Cedric921) [reply apps/web/app/interfaces/IDailyPlan.ts:12]: @Innocent-Akim will fix in next PR

**Comment by Innocent-Akim** [comment]
all right

Le mar. 16 juil. 2024 à 22:13, cedric karungu ***@***.***> a
écrit :

**Comment by evereq** [comment]
@Innocent-Akim please fix merge conflict

**Review by Cedric921 — APPROVED** [review]

**Pull request merged**